### PR TITLE
Matic (Sushi & Quick) with deployment

### PR DESCRIPTION
**Pull request by viacheslavvbogdanov**

### CODE REVIEW

**Comment by viacheslavvbogdanov** [comment]
Oracle for Matic network. Quick added instead of Uni

File: contracts/OracleBSC.sol [hunk 1]
```diff
@@ -10,7 +10,7 @@ import "./interface/mooniswap/IMooniswap.sol";
 import "./Governable.sol";
 import "@openzeppelin/contracts/utils/EnumerableSet.sol";
 
-import "hardhat/console.sol";
+//import "hardhat/console.sol";
```
Comment (belbix) [inline contracts/OracleBSC.sol:13]: remove console from the final code

Comment (belbix) [reply contracts/OracleBSC.sol:13]: ah it is BSC - better don't change anything

**Comment by viacheslavvbogdanov** [comment]
Added deployment script to MATIC. See [README.md]().

**Pull request merged**